packages/material/scss/map.scss

```diff
@@ -0,0 +1,14 @@
+@import "variables";
+@import "mixins";
+
+
+// Dependencies
+@import "common/all";
+@import "map/images/markers";
+@import "button";
+//@import "scroller";
```

Comment:
@joneff could you please explain why do we need the "scroller" in the Bootstrap theme? And do we need it here?